go.mod

```
@@ -1,72 +1,70 @@
 module github.com/kubearmor/kubearmor-client
 
-go 1.21
+go 1.21.0
```

```suggestion
go 1.21
```
Let's not ping go version